src/ScClient.ts

```diff
@@ -15,6 +15,12 @@ export interface Response<T = any> {
   status: boolean;
   event: boolean;
   payload: T;
+  errors: string | Error[];
+}
+
+export interface Error {
+  ref: number;
```

Comment:
`ref` name is a bit unclear as for me. Does it mean index or error code? Should error object hold this value?

Reply:
It is a key in json object. It uses in sc-server and py-sc-client. I name so, because it was there in the another command.  Many commands names and its fields names of sc-server do not always reflect the logic they do. I left issue in sc-machine about this problem https://github.com/ostis-ai/sc-machine/issues/177